### ISSUE TITLE
 Missing native symbol Java_jdk_internal_misc_Unsafe_putBoolean

### ISSUE THREAD

**Issue by jperedadnr** [javafxports/javafxmobile-plugin#57]
RuntimeException (without crash) parsing a json expression with booleans:

```WARNING: Failed to parse payload for list element: {"test":false}
java.lang.RuntimeException: Missing native symbol Java_jdk_internal_misc_Unsafe_putBoolean
	at java.lang.Throwable.fillInStackTrace(Unknown Source)
	at java.lang.Throwable.<init>(Unknown Source)
	at java.lang.Exception.<init>(Unknown Source)
	at java.lang.RuntimeException.<init>(Unknown Source)
	at jdk.internal.misc.Unsafe.putBoolean(Unknown Source)
	at jdk.internal.reflect.UnsafeBooleanFieldAccessorImpl.setBoolean(Unknown Source)
	at java.lang.reflect.Field.setBoolean(Unknown Source)
	at com.gluonhq.impl.cloudlink.client.data.metadata.SerializationUtils.readProperty(Unknown Source)
	at com.gluonhq.impl.cloudlink.client.data.metadata.ConnectMetadataClass.deserialize(Unknown Source)
	at com.gluonhq.impl.cloudlink.client.data.metadata.ConnectMetadataClass.deserialize(Unknown Source)
	at com.gluonhq.impl.cloudlink.client.data.GluonCloudListDataReaderImpl.next(Unknown Source)
	at com.gluonhq.connect.provider.DataProvider.lambda$retrieveList$25(Unknown Source)
	at com.gluonhq.connect.provider.DataProvider$$Lambda$7.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
```

**Comment by johanvos** [comment]
will be fixed in 2.0.22

**Comment by jperedadnr** [comment]
Indeed, it has been fixed
